pkg/totalaccountwatcher/totalaccountwatcher.go

```diff
@@ -46,14 +47,31 @@ type AccountWatcher struct {
 func initialize(client client.Client, watchInterval time.Duration) *AccountWatcher {
 	log.Info("Initializing the totalAccountWatcher")
 
+	cm, err := utils.GetOperatorConfigMap(client)
+	if err != nil {
+		log.Error(err, "Failed to get configmap")
+	}
+
+	awsRegion := awsv1alpha1.AwsUSEastOneRegion
+	fr, ok := cm.Data["fedramp"]
```

Comment:
This can all be replaced with `if isFedramp()` ? this would avoid getting the configmap again

Reply:
actually, we can replace that whole section with `config.GetDefaultRegion()` now